docs/content/hdfs.md

```diff
@@ -0,0 +1,166 @@
+---
+title: "HDFS Remote"
+description: "Remote for Hadoop Distributed Filesystem"
+---
+
+{{< icon "fa fa-globe" >}} HDFS
+-------------------------------------------------
+
+[HDFS](https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-hdfs/HdfsDesign.html) is a distributed file-system, part of [Apache Hadoop](https://hadoop.apache.org/) framework.
+
+Paths are specified as `remote:` or `remote:path/to/dir`.
+
```

Comment:
This manual page would hreatly benefit from a minimal stripped-down (no multiple OSes, no yarn or sparc examples) _hadoop getting started_, e.g. something like below:
```
sudo apt-get update # (or yum)
sudo apt-get install java
Please go to cloudera site and download yada-yada
tar xzf cloudera-standalone-installer.tgz
run installer...
blah...
configure rclone...
rclone copy ./data hdfs:/newdir1
hadoop fs -list newdir1
```

Reply:
in `fstest/testserver/images/test-hdfs` there is image for tests. is it ok to note about?

maybe with link to [SingleCluster](https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-common/SingleCluster.html)

Reply:
Yes, ok. If the image is present on docker hub then `docker run opts... repo/image:tag cmd...` is enough. If you provide only _Dockerfile_ then add either `docker build ...` or `docker-compose.yml` with a _build_ stanza and link it from docs.
While simply linking to cloudera _getting started_ would be _fine with me too_, I greatly appreciate your idea. A release on external vendor page will change with time while your documented image is under your control. It will give a basis to compare against in new bug/feature requests.

Reply:
updated.